client/wildcard/src/components/Combobox/Combobox.tsx

```diff
@@ -145,5 +147,27 @@ export const ComboboxOptionGroup = forwardRef((props, ref) => {
     )
 }) as ForwardReferenceComponent<'div', ComboboxOptionGroupProps>
 
-export { ReachComboboxOption as ComboboxOption }
+interface ComboboxOptionProps extends ReachComboboxOptionProps {
+    disabled?: boolean
+}
+
+export const ComboboxOption = forwardRef((props, ref) => {
+    const { disabled, children, ...attributes } = props
+    const context = useComboboxOptionContext()
+
+    if (disabled) {
+        return (
+            <li ref={ref} {...attributes}>
+                {typeof children === 'function' ? children(context) : children}
+            </li>
+        )
+    }
+
+    return (
+        <ReachComboboxOption ref={ref} {...attributes}>
+            {children}
```

Comment:
Does it make sense to account for callable `children` as we do above?

```suggestion
            {typeof children === 'function' ? children(context) : children}
```

Reply:
@valerybugakov ReachCombobox will handle it for us. Internally It does the same check with typeof children as we do above.